src/state/reducers/scigateway.reducer.test.tsx

```diff
@@ -280,6 +281,12 @@ describe('scigateway reducer', () => {
       ICATAuthProvider
     );
 
+    updatedState = ScigatewayReducer(state, loadAuthProvider('ldap-jwt'));
+
+    expect(updatedState.authorisation.provider).toBeInstanceOf(
+      LDAPJWTAuthProvider
+    );
```

Comment:
Could this be moved after the icat.anon one as its the same as the icat one above.